src/web/detectors/RotationGestureDetector.ts

```diff
@@ -143,24 +143,33 @@ export default class RotationGestureDetector
     return true;
   }
 
-  public getTimeDelta(): number {
-    return this.currentTime + this.previousTime;
+  public reset(): void {
+    this.keyPointers = [NaN, NaN];
+    this.isInProgress = false;
   }
 
-  public getAnchorX(): number {
-    return this.anchorX;
+  public get anchorX() {
+    return this._anchorX;
+  }
+  private set anchorX(value: number) {
+    this._anchorX = value;
   }
```

Comment:
Are setters actually needed here? 'm not sure if it should be possible to set the anchor externally on a gesture. Also applies to other places, do we need to create setters for **everything**?

Reply:
> 'm not sure if it should be possible to set the anchor externally on a gesture.

Well, it won't be set externally as the setter is private 😅 Also, I'm not sure if we want to write directly into backing field, so I'd leave that 😅

Reply:
What's the use case for a setter that doesn't do anything besides setting the backing field? 
Isn't it equivalent to just having an `anchorX` property available, and removing the `_anchorX` backing field?
I believe this PR was meant to migrate `getSomething` functions into `something` getters, but converting every existing regular property into a `getter`+`setter` doesn't seem useful at all.

Reply:
After some thinking, I think I agree with @latekvo. Having private setters that essentially don't do anything may do more harm than good just by being there (case in point, I missed the `private` when typing the first comment). I'm not sure there's anything wrong with setting the backing field directly - isn't this what we're doing right now but with a `getX` function?

Reply:
> (...) converting every existing regular property into a `getter`+`setter` doesn't seem useful at all.

It is not converting _every existing property_, as you've already mentioned below 😉

> Isn't it equivalent to just having an anchorX property available, and removing the _anchorX backing field? 

It would be equivalent to making it `public` which I don't think we want to do.


> I'm not sure there's anything wrong with setting the backing field directly - isn't this what we're doing right now but with a getX function?

Basically yes, if you prefer to do so, I can change it

Reply:
Yeah, I think it would make sense. The diff would be the smallest (I think) this way and we can always iterate further on this.

Reply:
Done in [336f744](https://github.com/software-mansion/react-native-gesture-handler/pull/3263/commits/336f74490fd1ef96ea0596741a27060c91015c26)